src/translations.js

```diff
@@ -35,9 +43,24 @@ function getSortedObject(object) {
 
 function fillMissingKeys(baseObject, object) {
   Object.keys(baseObject).forEach(baseKey => {
-    if (!object[baseKey]) {
+    if (typeof object[baseKey] === 'undefined') {
       object[baseKey] = '';
+      debug(`ADDED KEY: ${baseKey}`);
     }
   });
   return object;
 }
+
+function deleteExtraKeys(baseObject, object) {
+  const baseKeys = Object.keys(baseObject) || [];
```

Comment:
Object.keys always returns array. Not sure about baseObject always being an object though.